templates/mail/geofenceExit.vm

```diff
@@ -4,7 +4,7 @@
 <body>
 Device: $device.name<br>
 Has exited geofence: $geofence.name<br>
-Point: <a href="https://www.openstreetmap.org/?mlat=$position.latitude&mlon=$position.longitude#map=16/$position.latitude/$position.longitude">#if($position.address)$position.address#{else}$position.latitude°, $position.longitude°#end</a><br>
 Time: $event.serverTime<br>
+Point: <a href="$webUrl?eventId=$event.id">#if($position.address)$position.address#{else}$position.latitude°, $position.longitude°#end</a><br>
```

Comment:
Can it be used for "if" as well to make it consistent? I think it would look better.